webui/react/src/components/TaskList.tsx

```diff
@@ -463,14 +466,15 @@ const TaskList: React.FC<Props> = ({ workspace }: Props) => {
         dataIndex: 'user',
         defaultWidth: DEFAULT_COLUMN_WIDTHS['user'],
         filterDropdown: userFilterDropdown,
```

Comment:
ideally we could make it where we provide a loading state to the dropdown, but i dont think it's worth it when we're in the process of switching tables